changes.d/6431.fix.md

```diff
@@ -0,0 +1 @@
+The `cycle point format` was imposing an undesirable constraint on `wall_clock` offsets, this has been fixed.
```

Comment:
```suggestion
Allowed `wall_clock` offsets to use smaller precision than the `cycle point format` (e.g. `wall_clock(offset="PT1M")` in combination with `cycle point format = CCYY`).
```

Reply:
Well technically they were capable of it before, but some change broke it.

Reply:
Approved, but I prefer @oliver-sanders wording here, in terms of clarity for users.